src/main/java/org/prebid/server/bidder/mgid/model/BidExtResponse.java

```diff
@@ -0,0 +1,11 @@
+package org.prebid.server.bidder.mgid.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@Value
+@AllArgsConstructor(staticName = "of")
+public class BidExtResponse {
+
+    String crtype;
```

Comment:
I think it is better to use `org.prebid.server.proto.openrtb.ext.response.BidType` type for this filed.
I now, we discussed this and decided to use `String` but i see the issue with `native` value.